resources/todomvc/big-dom-generator/src/tree-generator.js

```diff
@@ -86,6 +69,15 @@ export const generateTreeHead = ({ listWeight, expandableItemWeight, nonExpandab
                         treeNodes.push(...currentNode.children);
                     }
                     break;
+                case "nonExpandableItem":
+                    if (random.coin(PROBABILITY_OF_HAVING_CHILDREN)) {
+                        currentNode.type = "expandableItem";
+                        currentNode.children = [{ type: "expandableItem", children: [] }];
```

Comment:
Randomly decide if the child will be expandable or nonExpandable.  Make sure you modify the weigtht too.

Reply:
@lpardosixtosMs I recall there was also a comment you wanted me to add here?